Library/Homebrew/utils/repology.rb

```diff
@@ -0,0 +1,84 @@
+# frozen_string_literal: true
+
+require "utils/curl"
+require "utils/versions"
+
+require "formula_info"
+
+module RepologyParser
+  module_function
+
+  MAX_PAGINATION = 15
+
+  def query_api(last_package_in_response = "")
+    last_package_in_response += "/" unless last_package_in_response.empty?
+
+    url = "https://repology.org/api/v1/projects/#{last_package_in_response}?inrepo=homebrew&outdated=1"
+    ohai "Calling API #{url}" if Homebrew.args.verbose?
```

Comment:
```suggestion
    puts "Calling API #{url}" if Homebrew.args.verbose?
```

Though I'm not sure if passing `--verbose` will emit the `curl` URL anyway, so it might be good to check to see if this is even needed.